internal/service/streamconnection/data_source_stream_connection_test.go

```diff
@@ -0,0 +1,83 @@
+package streamconnection_test
+
+import (
+	"fmt"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-testing/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/mongodb/terraform-provider-mongodbatlas/internal/testutil/acc"
+)
+
+func TestAccStreamDSStreamConnection_kafkaPlaintext(t *testing.T) {
+	var (
+		orgID          = os.Getenv("MONGODB_ATLAS_ORG_ID")
```

Comment:
[nit] not need to address it here. I am wondering if we should define `MONGODB_ATLAS_ORG_ID` and other env vars in a common file and then import them. In this way, if we change the env var we just need to update it in one place.

Reply:
could remove some boilerplate for sure, on the other hand it does help to see want env variables are being used for each test so dont have a strong opinion